tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -10,6 +10,105 @@ exports[`access_hidden_namespace.w 1`] = `
 
  
  
+Tests 1 failed (1)
+Test Files 1 failed (1)
+Duration <DURATION>"
+`;
+
+exports[`access_modifiers.w 1`] = `
+"error: Cannot access protected member protected_field of Foo
+   --> ../../../examples/tests/invalid/access_modifiers.w:64:9
+   |
+64 | log(foo.protected_field);
+   |         ^^^^^^^^^^^^^^^ Cannot access protected member protected_field of Foo
```

Comment:
Should we put names of user identifiers in quotes or backticks to signal they're not part of the error message? e.g.
```suggestion
   |         ^^^^^^^^^^^^^^^ Cannot access protected member "protected_field" of "Foo"
```

I don't think anyone has proposed a style guide yet (https://github.com/winglang/wing/issues/1746) but I think the compiler already does this for some other error messages